docs/test acceleration/getting-started/pytest.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: default
+title: Pytest
+nav_order: 1
+parent: Getting started
+grand_parent: Test acceleration
+permalink: /test-acceleration/getting-started/pytest
+---
+
+# Try in pytest
+{:.no_toc}
```

Comment:
Not sure what's going on with the table of contents here

![image](https://user-images.githubusercontent.com/82620675/130520962-0b08c9c1-1c18-4b20-8dac-fa5879433fe5.png)

Reply:
Fixed